Client.py

```diff
@@ -226,19 +227,19 @@ def algorithm(
                     _this_paragraph_dictionary[noun][answer] = dict()
             return
         else:
-            data = [sent.text for sent in sentences]
+            data = [sent.text for sent in number_of_sentences]
```

Comment:
Should it be `range(number_of_sentences)`?

Reply:
this is to get sentence fixed after spacy change